### DISCUSSION TITLE
How do I nudge the entire animation to the right?

### DISCUSSION THREAD

**Discussion by budaloops** [keiwando/toonsquid#689] [Q&A] [answered]
Realized that the shape of the letter Y makes the composition look off-center so I need to adjust it. But I would have to move all 16 of my animated layers individually. Is there an easier way?

![IMG_3582](https://github.com/user-attachments/assets/09de39a4-e0b2-4a70-b2b6-45a61338c8fc)

**Comment by Cat-Butler** [comment, marked as answer]
Attach them to a Transform layer in the timeline Hierarchy and nudge the Transform layer as a frame 1 keyframe?

**Comment by budaloops** [comment]
It worked! 🎖️ Thanks for taking the time to answer a simple problem
